website/src/pages/login.css.ts

```diff
@@ -0,0 +1,40 @@
+import { style } from "@vanilla-extract/css"
+import { button, centeredColumn, edgePadded } from "src/sprinkles.css"
+
+export const loginHeader = style({
+  display: "flex",
+  justifyContent: "flex-end",
+  flex: 5,
+  padding: "8px",
+})
+
+export const logoutPopover = style({
+  display: "flex",
+  flexDirection: "column",
+  backgroundColor: "white",
+  padding: "16px",
+  borderRadius: "10%",
```

Comment:
Avoid percentages for this, check the `sprinkles.css.ts` file for some value presets we have. In there, we have `vspace` for vertical padding/margins, `hspace` for horizontal padding/margins, `radii` for `borderRadius` values, `colors` for `color` or `backgroundColor`, etc. I'd replace many of the values you have here with some of those. i.e. `borderRadius: radii.medium` perhaps. Feel free to add more values to those objects as you see fit though!